### DISCUSSION TITLE
Problem with extensions.

### DISCUSSION THREAD

**Discussion by cacuestai** [coder/code-server#4196] [Q&A]
Hello:

On my Ubuntu machine, the application does not load the extensions that I install.

Thanks for the suggestions,

Carlos

**Comment by lg-nineoneninethreenine** [comment]
I think you must wait for the extensions to load, like, whenever I install an extension, I need to wait. Even you should. What version of code-server are you using?

**Reply by cacuestai** [reply]
Thanks for answering. This did not happen to me in versions prior to 3.11, but it did with the new ones, even with the 3.12 that I currently use. I have tried on two computers, with several browsers and both have the same problem.

Cheers

**Reply by lg-nineoneninethreenine** [reply]
Expected this, I think @jsjoeio has the right answer.

**Reply by cacuestai** [reply]
Could you please provide me with the specific link of your answer?

**Reply by jsjoeio** [reply]
Hmm...I'm not sure what exactly is happening. Can you please open a [bug report](https://github.com/cdr/code-server/issues/new?assignees=&labels=&template=bug-report.md&title=) and fill everything out? If you can include screenshots or a video too, that would help. Thanks!
